config/services.yaml

```diff
@@ -124,4 +124,10 @@ services:
 
     App\Domain\Strava\Ftp\ImportFtp\FtpValuesFromEnvFile:
         factory: [ 'App\Domain\Strava\Ftp\ImportFtp\FtpValuesFromEnvFile', fromString ]
-        arguments: [ '%env(string:FTP_VALUES)%' ]
\ No newline at end of file
+        arguments: [ '%env(string:FTP_VALUES)%' ]
+
+    App\Domain\Strava\Ftp\EFtpRepository:
+        factory: [ 'App\Domain\Strava\Ftp\InMemoryEFtpRepository', from ]
+        arguments: [ 
+            '%env(int:default::CALCULATE_EFTP_BASED_ON_LAST_NUMBER_OF_MONTHS)%'
```

Comment:
I'd rather have this in a dedicated value object and inject that value object into the repository with auto-wiring.